### DISCUSSION TITLE
Pytest ends with errors about dynamic-prompts

### DISCUSSION THREAD

**Discussion by tmprabubiz** [adieyal/sd-dynamic-prompts#598] [Q&A] [answered]
1. Reinstalled sd-dynamic-prompts using git pull, then tried magic prompts to download all the models, that time I could see the prompts below the image. But...

2. Made the settings as said in the page including create a prompts file. But now I could neither see any prompt below the image, nor in the csv file created, unfortunately infinite image browser too gives only generation details. What should I do to get the prompts.

3. Today read about pytest, installed and ran pytest multiple times to find many warnings and errors. After new installation of sd-dynamic-prompts, the  pytest, ended with warnings and errors, the first four were about SD prompting. Please find below and suggest a remedy. I don't know coding

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=============================================== short test summary info ===============================================
ERROR extensions/dynamic-prompts/tests/prompts/test_attention_generator.py
ERROR extensions/dynamic-prompts/tests/prompts/test_frozenprompt_generator.py
ERROR extensions/dynamic-prompts/tests/prompts/test_generator_builder.py
ERROR extensions/dynamic-prompts/tests/prompts/test_helpers.py
ERROR extensions/dynamic-prompts/tests/prompts/ui/test_pnginfo_saver.py
ERROR extensions/dynamic-prompts/tests/prompts/ui/test_prompt_writer.py
ERROR extensions/sd-webui-aspect-ratio-helper/tests/_util_test.py
ERROR extensions/sd-webui-controlnet/annotator/leres/pix2pix/options/test_options.py
ERROR extensions/sd-webui-controlnet/annotator/mmpkg/mmseg/datasets/pipelines/test_time_aug.py
ERROR extensions/sd-webui-controlnet/annotator/oneformer/detectron2/modeling/test_time_augmentation.py
ERROR extensions/sd-webui-controlnet/tests/annotator_tests/openpose_tests/body_test.py
ERROR extensions/sd-webui-controlnet/tests/annotator_tests/openpose_tests/detection_test.py
ERROR extensions/sd-webui-controlnet/tests/annotator_tests/openpose_tests/json_encode_test.py
ERROR extensions/sd-webui-controlnet/tests/annotator_tests/openpose_tests/openpose_e2e_test.py
ERROR extensions/sd-webui-controlnet/tests/cn_script/batch_hijack_test.py
ERROR extensions/sd-webui-controlnet/tests/cn_script/cn_script_test.py
ERROR extensions/sd-webui-controlnet/tests/cn_script/utils_test.py
ERROR extensions/sd-webui-controlnet/tests/external_code_api/external_code_test.py
ERROR extensions/sd-webui-controlnet/tests/external_code_api/importlib_reload_test.py
ERROR extensions/sd-webui-controlnet/tests/external_code_api/script_args_test.py
ERROR extensions/sd-webui-controlnet/tests/web_api/control_types_test.py
ERROR extensions/sd-webui-controlnet/tests/web_api/detect_test.py
ERROR extensions/sd-webui-controlnet/tests/web_api/img2img_test.py
ERROR extensions/sd-webui-controlnet/tests/web_api/txt2img_test.py
ERROR extensions/sd-webui-faceswaplab/tests/test_api.py
ERROR repositories/stable-diffusion-stability-ai/scripts/tests/test_watermark.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 26 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================== 103 warnings, 26 errors in 10.05s ==========================================

**Comment by adieyal** [comment, marked as answer]
I'm not sure that I understand your question. Is dynamic prompts working at all or is it broken only when you use dynamic prompts?

**Reply by tmprabubiz** [reply]
Thank You. 
The error rectified itself. The prompts generated by "prompt magic" was not found earlier. 
Now the prompts are showing up. 
Thank You again.
